toptal/davinci-github-actions
# feat: add report-missing-dependecy action

[FX-3300]

### Description

Add new action for monorepo that report changed packages which are not in changeset.
The difference is reported as a PR comment.

### How to test

- you can check out the [Picasso PR](https://github.com/toptal/picasso/pull/3286#issuecomment-1350664508), push some changes to see how it behaves

### Development checks


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>


[FX-3300]: https://toptal-core.atlassian.net/browse/FX-3300?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
